vine/src/structures/diag.rs

```diff
@@ -293,6 +295,8 @@ diags! {
     ["unused variable"]
   UnusedItem
     ["unused item"]
+  BadTestAttr
+    ["the `#[test]` attribute can only be applied to an fn"]
```

Comment:
Can you change this and `BadFramelessAttr` to say `a function`? I don't know why I wrote it as `an fn`...